src/middlewares/verifyToken.js

```diff
@@ -27,18 +27,25 @@ export default (request, response, next) => {
 
   jwt.verify(token, secret, async (error, decoded) => {
     if (error) {
-      const message = (error.name === 'TokenExpiredError') ? 'token expired, you have to be signed in to continue' : 'you have to be signed in to continue';
-      return responseMessage(response, 401, { error: message });
-    }
-    try {
-      const user = await findUser(decoded.id);
-      if (!user) {
-        return responseMessage(response, 404, { error: 'user not found' });
+      const message = (error.name === 'TokenExpiredError') ? 'token expired' : 'invalid token';
+      responseMessage(response, 401, { error: message });
+    } else {
+      try {
+        const isBlacklisted = await findBlacklistedToken(token);
```

Comment:
Don't you think blaslistedToken is a better variable name in this case?

Reply:
Yeah. I think blacklistedToken is more descriptive. Thanks